test/models/userRefs.ts

```diff
@@ -5,7 +5,7 @@ export class UserRef {
   public master?: Ref<UserRef>;
 
   @arrayProp({ ref: UserRef, default: [] })
-  public subAccounts!: Ref<UserRef>[];
+  public subAccounts?: Ref<UserRef>[];
```

Comment:
why did this get changed?

Reply:
Causes a ton of errors in `typeguards.test.ts`.